ee/localserver/krypto-ec-middleware.go

```diff
@@ -68,13 +80,19 @@ func (e *kryptoEcMiddleware) Wrap(next http.Handler) http.Handler {
 		// reusing it a bunch. However, it will fail if the clocks are too far out of sync.
 		timestampDelta := time.Now().Unix() - challengeBox.Timestamp()
 		if timestampDelta > timestampValidityRange || timestampDelta < -timestampValidityRange {
+			span.SetAttributes(attribute.Int64("timestamp_delta", timestampDelta))
+			span.SetStatus(codes.Error, "timestamp is out of range")
+
 			level.Debug(e.logger).Log("msg", "timestamp is out of range", "delta", timestampDelta)
```

Comment:
One thing I want to call out here, is that I think the _eventual_ convergence here is that we merge logging and spans. And I have no idea what that looks like